configuration/Makefile

```diff
@@ -0,0 +1,15 @@
+stop:
+	docker-compose down
+
+configuration:
+	docker-compose up
+
+status:
+	docker-compose exec etcd1 etcdctl endpoint status
```

Comment:
now that it is an `healthcheck` this should just be `docker-compose ps`?

Reply:
The health check is different to the status though :thinking: ?

Reply:
Didn't notice that, then good call.